src/widgets/merch/ui/merch.test.tsx

```diff
@@ -9,32 +10,32 @@ describe('Merch', () => {
   });
 
   it('renders the title correctly', () => {
-    const titleElement = screen.getByText('RS merch');
+    const titleElement = screen.getByText(MOCKED_MERCH_DATA.title);
 
     expect(titleElement).toBeVisible();
   });
 
   it('renders the subtitle correctly', () => {
-    const subtitleElement = screen.getByText('Are you an RS sloth fan and looking for RS merch?');
+    const subtitleElement = screen.getByText(MOCKED_MERCH_DATA.subtitle);
 
     expect(subtitleElement).toBeVisible();
   });
 
   it('renders the paragraph correctly', () => {
-    const paragraphText = screen.getByText(/The wait is almost over/i);
+    const paragraphText = screen.getByText(new RegExp(MOCKED_MERCH_DATA.paragraph, 'i'));
 
     expect(paragraphText).toBeVisible();
   });
 
   it('renders the merch "Discover merch assets" button with correct href', () => {
-    const buttonElement = screen.getByRole('link', { name: /Discover merch assets/i });
+    const buttonElement = screen.getByRole('link', { name: new RegExp(MOCKED_MERCH_DATA.buttonText, 'i') });
 
     expect(buttonElement).toBeVisible();
-    expect(buttonElement).toHaveAttribute('href', 'https://sloths.rs.school/');
+    expect(buttonElement).toHaveAttribute('href', MOCKED_MERCH_DATA.buttonLink);
   });
 
   it('renders the image with alt text', () => {
-    const imageElement = screen.getByAltText('speakers-wanted');
+    const imageElement = screen.getByAltText(MOCKED_MERCH_DATA.imageAltText);
```

Comment:
Reviewers asked me to add `data-testid` to images to find the element in tests, and below in the test to check the visibility and content of attributes `.toHaveAttribute('alt', ...)` and `.toHaveAttribute('src', ...);` Can we do it here too?

Reply:
Done